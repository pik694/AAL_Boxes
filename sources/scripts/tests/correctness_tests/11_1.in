11
8 99 33
6 38 17
7 6 15
89 27 57
96 1 38
41 42 6
96 89 37
2 74 60
60 97 92
37 3 57
4 94 74

