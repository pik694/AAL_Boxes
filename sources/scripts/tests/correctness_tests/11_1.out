2 4 1 0 9 7 10 5 3 6 8 
1076192->600005
